### PR TITLE
Production- adding how to docs

### PR DESCRIPTION
**Pull request by sulthan-ahmed**
Because it's not straight forward and it'll save ACP folks a lot of time

### CODE REVIEW

File: docs/how-to-docs/production.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+### Set up a prod token <a name="prod-token"></a>
+
+The following steps in this section can only be done by an administrator
+* Go to the [platformhub](https://hub.acp.homeoffice.gov.uk)
+* Click projects -> modern slavery -> details
+* Scroll to the top and click on `KUBE USER TOKENS`
+* And then search for your name in the drop-down
+* And then click `create a new kubernetes user token for this user`
+* Add the user as read-only
+* Escalate priveledges to two hours, you'll need to do this everytime you are on.  If you're not an administrator then you'll have to ask your admin
```
Comment (zuzak) [inline docs/how-to-docs/production.md]:
```suggestion
* Escalate privileges to two hours, you'll need to do this everytime you are on.  If you're not an administrator then you'll have to ask your admin
```

**Comment by sulthan-ahmed** [comment]
thanks @zuzak

File: docs/how-to-docs/production.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+  2. [add the cluster to your kube file](#add-token)
+  3. [add the token to your kube file](#prod-cluster)
+  4. [connect to the prod vpn](#prod-vpn)
+  5. [drone](#prod-drone)
+
+### Set up a prod token <a name="prod-token"></a>
+
+The following steps in this section can only be done by an administrator
+* Go to the [platformhub](https://hub.acp.homeoffice.gov.uk)
+* Click projects -> modern slavery -> details
```
Comment (chrisns) [inline docs/how-to-docs/production.md]: ~~modern slavery~~ [your project]

File: docs/how-to-docs/production.md [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+
+### Add the user token to your kube file <a name="add-token"></a>
+
+* Copy the token you generated above
+* Go to your kube config file, usually located in `~./kube/config`
+* Add the following to your users
+
+```
+  users:
+  - name: acp-prod
```
Comment (chrisns) [inline docs/how-to-docs/production.md]: iirc user tokens are scoped to the project in acp? if so you might want to name them more specifically

File: docs/how-to-docs/production.md [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+    user:
+      token: xxxxx
+```
+
+* add a context to your config like so
+
+```
+- context:
```
Comment (chrisns) [inline docs/how-to-docs/production.md]: you can do this more easily with
```kubectl config set-context```

File: docs/how-to-docs/production.md (hunk 3 above)
Comment (chrisns) [reply docs/how-to-docs/production.md]: you can do this more easily with
```kubectl config set-context```

File: docs/how-to-docs/production.md [hunk 5]
```diff
@@ -0,0 +1,72 @@
...
+### Production vpn <a name="prod-vpn"></a>
+
+Now connect to the acp-prod vpn.  If you can't see access you'll need to raise a ticket with ACP
+https://support.acp.homeoffice.gov.uk/servicedesk/customer/portal/1/create/90
+
+Now you should be able to use kubectl like normal.  Remember to escalate your priviledges.  If you are not admin, you'll have to ask your administrator to escalate it for you
+
+### Production- Drone deployment <a name="prod-drone"></a>
+
+Setting up deployments as the same as deployment in not-prod but you need to add a robot and give it priviledges.
```
Comment (chrisns) [inline docs/how-to-docs/production.md]: ~~Setting up deployments as the same~~ Setting up deployments is the same

Comment (chrisns) [reply docs/how-to-docs/production.md]: ~~priviledges~~ privileges

**Review by chrisns — CHANGES_REQUESTED** [review]

**Comment by chrisns** [comment]
you might also want to make reference to the use of https://github.com/ahmetb/kubectx for switching between contexts and (self promotion) https://github.com/chrisns/kubectl-passman for not just putting all your creds in clear text

**Pull request closed without merging**